src/providers/github/Github.ts

```diff
@@ -48,14 +48,22 @@ export class Github {
   // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
   async getRepo() {
     try {
-      return await this.octokit.repos.get({
-        owner: this.owner,
-        repo: this.repo
-      });
+      const repoResponse = await this.octokit.request(
```

Comment:
why are you using `oktokit.request` instead of `oktokit.rest` 

According to docs https://github.com/octokit/octokit.js?tab=readme-ov-file#rest-api both serve the same purpose but `oktokit.rest` looks better visually IMO. Since is typed you will only be allowed to do a request defined in the library.

Reply:
I used it in this way because it is defined like that in the Github REST API Docs, but I agree it looks cleaner with `rest`: https://docs.github.com/en/rest/branches/branches?apiVersion=2022-11-28#list-branches